src/index.html

```diff
@@ -6,13 +6,64 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="/src/styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="/src/images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A) "
+        class="card__image"
+      />
+
+      <header class="card__text-container">
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+      </header>
+
+      <div
+        class="stars"
+        data-qa="stars"
+      >
+        <section class="stars__container">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+        </section>
+
+        <span class="stars__reviews">Reviews: 5</span>
+      </div>
+
+      <div class="card__price-container">
+        <span class="card__price-label">Price:</span>
+        <span class="card__price-value">$2,199</span>
+      </div>
+
+      <button
+        href="#"
```

Comment:
The `href` attribute is not valid for a `button` element. If you want to navigate to a different page when the button is clicked, consider using a `button` element with an `onclick` event handler in JavaScript, or use an `a` (anchor) tag instead.